daml-foundations/daml-ghc/ghc-compiler/src/DA/Daml/GHC/Compiler/Convert.hs

```diff
@@ -1245,9 +1245,9 @@ convertTyCon env t
             "Party" -> pure TParty
             "Date" -> pure TDate
             "Time" -> pure TTimestamp
+            "TextMap" -> pure (TBuiltin BTMap)
             _ -> defaultTyCon
     | isBuiltinName "Optional" t = pure (TBuiltin BTOptional)
```

Comment:
Should this optional not be one further up? Isn't it in DA.Internal.LF? Should it be?

Reply:
Given that we (should) erase during conversion to DAML-LF as well, it would make sense to have it in `DA.Internal.LF`. I'll move it in a separate PR.